Resources/config/api.xml

```diff
@@ -31,6 +31,54 @@
             <argument type="service" id="api.router" />
             <argument type="service" id="property_accessor" />
         </service>
+
+        <service id="api.format_negotiator" class="Negotiation\FormatNegotiator" public="false" />
+
+        <!-- Event listeners -->
+
+        <service id="api.listener.request.resource_type" class="Dunglas\ApiBundle\EventListener\ResourceTypeRequestListener">
+            <argument type="service" id="api.resource_collection" />
+
+            <tag name="kernel.event_listener" event="kernel.request" method="onKernelRequest" />
+        </service>
+
+        <service id="api.listener.request.format" class="Dunglas\ApiBundle\EventListener\FormatRequestListener">
+            <argument type="service" id="api.format_negotiator" />
+            <argument>%api.supported_formats%</argument>
+
+            <tag name="kernel.event_listener" event="kernel.request" method="onKernelRequest" />
+        </service>
+
+        <service id="api.listener.view.validation" class="Dunglas\ApiBundle\EventListener\ValidationViewListener">
+            <argument type="service" id="validator" />
+
+            <tag name="kernel.event_listener" event="kernel.view" method="onKernelView" priority="20" />
```

Comment:
Why 20 ? I am saying that because for sf standard edition that we use for `api-platform`, There is only one other listener for the `kernel.view`with a 0 default priority
<img width="938" alt="capture d ecran 2015-07-28 a 09 44 08" src="https://cloud.githubusercontent.com/assets/911705/8926151/b31d108c-350d-11e5-83e9-fd56cd7789be.png">